### PR TITLE
docs: add EU vision page - INNO-1775

### PR DESCRIPTION
**Pull request by yhuard**
Reference: https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/1.+Vision+-+ECL+EU+2

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit d0c50f7658e23fb8f795bed68f0c28f59f3ffd9b

https://deploy-preview-1306--europa-component-library.netlify.com

File: src/website/src/pages/eu/about/vision/index.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Vision
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+## Your gateway to the European Union
+
+<Paragraph size="lead">
```
Comment (planctus) [inline src/website/src/pages/eu/about/vision/index.mdx:10]: I see in the wiki they expect the font-size of the paragraph to change in mobile...

File: src/website/src/pages/eu/about/digital-design-principles/index.mdx [hunk 2]
```diff
@@ -14,31 +14,50 @@ import { Paragraph } from '@ecl/website-components';
   findings.
 </Paragraph>
 
+<div style={{borderLeft: '4px solid #0E47CB', marginTop: '3rem', paddingLeft: '1rem'}}>
+
 ## Connecting with people
```
Comment (planctus) [inline src/website/src/pages/eu/about/digital-design-principles/index.mdx:19]: These h2s they get the border bottom that i don't see in the specs but the border is in place in the mocks for the "standalone" h2s, apparently... These are also black in the mock, while the others are grey. I don't know if we are supposed to reproduce all of these design specs or inconsistencies, depending on the way we look at them..

File: src/website/src/pages/eu/about/vision/index.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Vision
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+## Your gateway to the European Union
```
Comment (planctus) [inline src/website/src/pages/eu/about/vision/index.mdx:8]: I see you set the "subtitle" as in the text-box in the wiki, while the mock is different, confusing.. as it is confusing to see in mobile a different heading for the section in the "specs" but yeah, guess you're right :)

**Review by planctus — COMMENTED** [review]
Not sure @yhuard , you can read the comments but i guess it's something you are already perfectly aware of, so it's good for me and if we don't have to reproduce everything exactly as we find it in the mocks then we just merge it as it is.

**Review by planctus — APPROVED** [review]

**Pull request merged**